### PR TITLE
Add wwn and serial checking for spapr_vscsi:

### PR DESCRIPTION
**Pull request by zhenyzha**
Modify the configure file to support checking wwn and serial options
in ppc.

ID: 1769669
Signed-off-by: Zhenyu Zhang <zhenyzha@redhat.com>

### CODE REVIEW

**Comment by zhenyzha** [comment]
Test result:
 (3/6) Host_RHEL.m8.u0.qcow2.spapr_vscsi.up.virtio_net.Guest.RHEL.8.1.0.ppc64le.io-github-autotest-qemu.device_option_check.blk_wwn_check.wwn_disk.default_drive_format: PASS (104.64 s)
 (4/6) Host_RHEL.m8.u0.qcow2.spapr_vscsi.up.virtio_net.Guest.RHEL.8.1.0.ppc64le.io-github-autotest-qemu.device_option_check.blk_wwn_check.wwn_disk.special_drive_format: PASS (100.35 s)
 (5/6) Host_RHEL.m8.u0.qcow2.spapr_vscsi.up.virtio_net.Guest.RHEL.8.1.0.ppc64le.io-github-autotest-qemu.device_option_check.blk_wwn_check.wwn_cdrom.default_drive_format: PASS (101.55 s)
 (6/6) Host_RHEL.m8.u0.qcow2.spapr_vscsi.up.virtio_net.Guest.RHEL.8.1.0.ppc64le.io-github-autotest-qemu.device_option_check.blk_wwn_check.wwn_cdrom.special_drive_format: PASS (101.91 s)

**Comment by zhenyzha** [comment]
@YongxueHong @ngu-niny Please help to review, thanks.

File: qemu/tests/cfg/device_option_check.cfg [hunk 1]
```diff
@@ -41,6 +41,8 @@
                         force_drive_format = ide-drive
                     virtio_scsi:
                         force_drive_format = scsi-disk
+                    spapr_vscsi:
```
Comment (ngu-niny) [inline qemu/tests/cfg/device_option_check.cfg]: Seems the 'special_drive_format' test is not necessary since the default 'drive_image' is 'scsi-hd'.

Comment (zhenyzha) [reply qemu/tests/cfg/device_option_check.cfg]: Corrected as scsi-disk, thanks for reminding.

**Comment by zhenyzha** [comment]
Test result:
(1/3) Host_RHEL.m8.u0.qcow2.spapr_vscsi.up.virtio_net.Guest.RHEL.8.1.0.ppc64le.io-github-autotest-qemu.device_option_check.blk_wwn_check.wwn_disk.default_drive_format: PASS (133.59 s)
 (2/3) Host_RHEL.m8.u0.qcow2.spapr_vscsi.up.virtio_net.Guest.RHEL.8.1.0.ppc64le.io-github-autotest-qemu.device_option_check.blk_wwn_check.wwn_disk.special_drive_format: PASS (125.45 s)
 (3/3) Host_RHEL.m8.u0.qcow2.spapr_vscsi.up.virtio_net.Guest.RHEL.8.1.0.ppc64le.io-github-autotest-qemu.device_option_check.blk_wwn_check.wwn_cdrom.default_drive_format: PASS (145.33 s)

**Comment by zhenyzha** [comment]
@ngu-niny Please help to review, thanks.

**Comment by zhenyzha** [comment]
Test result:
 (1/4) Host_RHEL.m8.u0.qcow2.spapr_vscsi.up.virtio_net.Guest.RHEL.8.1.0.ppc64le.io-github-autotest-qemu.device_option_check.blk_wwn_check.wwn_disk.default_drive_format: PASS (127.07 s)
 (2/4) Host_RHEL.m8.u0.qcow2.spapr_vscsi.up.virtio_net.Guest.RHEL.8.1.0.ppc64le.io-github-autotest-qemu.device_option_check.blk_wwn_check.wwn_disk.special_drive_format: PASS (114.94 s)
 (3/4) Host_RHEL.m8.u0.qcow2.spapr_vscsi.up.virtio_net.Guest.RHEL.8.1.0.ppc64le.io-github-autotest-qemu.device_option_check.blk_wwn_check.wwn_cdrom.default_drive_format: PASS (117.82 s)
 (4/4) Host_RHEL.m8.u0.qcow2.spapr_vscsi.up.virtio_net.Guest.RHEL.8.1.0.ppc64le.io-github-autotest-qemu.device_option_check.blk_serial_check: PASS (126.35 s)

**Comment by ngu-niny** [comment]
LGTM.

Test PASS:
# ./scan_results.py 
CaseName                                 Status          StartTime       EndTime         Result     TimeElapsed
'1-Host_RHEL.m8.u1.qcow2.spapr_vscsi.up.virtio_net.Guest.RHEL.8.1.0.ppc64le.io-github-autotest-qemu.device_option_check.blk_wwn_check.wwn_disk.default_drive_format' Finshed 2019-11-13 02:20:08 2019-11-13 02:21:48 PASS 99.759097
'2-Host_RHEL.m8.u1.qcow2.spapr_vscsi.up.virtio_net.Guest.RHEL.8.1.0.ppc64le.io-github-autotest-qemu.device_option_check.blk_wwn_check.wwn_disk.special_drive_format' Finshed 2019-11-13 02:21:48 2019-11-13 02:23:07 PASS 79.139462
'3-Host_RHEL.m8.u1.qcow2.spapr_vscsi.up.virtio_net.Guest.RHEL.8.1.0.ppc64le.io-github-autotest-qemu.device_option_check.blk_wwn_check.wwn_cdrom.default_drive_format' Finshed 2019-11-13 02:23:07 2019-11-13 02:24:39 PASS 91.930635
'4-Host_RHEL.m8.u1.qcow2.spapr_vscsi.up.virtio_net.Guest.RHEL.8.1.0.ppc64le.io-github-autotest-qemu.device_option_check.blk_serial_check' Finshed 2019-11-13 02:24:39 2019-11-13 02:26:10 PASS 90.704596
Summary: 
Finshed=4, PASS=4

Acked-by: Nini Gu <ngu@redhat.com>

**Pull request merged**